pkg/util/filesystem/permission_windows_test.go

```diff
@@ -0,0 +1,80 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package filesystem
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"unsafe"
+
+	"github.com/stretchr/testify/require"
+
+	"golang.org/x/sys/windows"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
+)
+
+func TestRemoveAccessToOtherUsers(t *testing.T) {
+	getDataDogUserSid = winutil.GetSidFromUser
+
+	sid, _, _, err := windows.LookupSID("", "ddagentuser")
+	require.NoError(t, err)
+	t.Log(sid)
```

Comment:
what's this for?

Reply:
was just checking something. meant to delete